etc/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -167,6 +167,22 @@
     regexp: '^(Subsystem.*sftp.*)'
     replace: '#\1'
 
+#
+# Remove the last-login message when ssh-ing.
+#
+- replace:
+    dest: /etc/ssh/sshd_config
+    regexp: '#PrintLastLog yes'
```

Comment:
I think we want to use something like this: `^#?PrintLastLog` .. This way, it'll modify the line if the field is commented out or not, and regardless of the current setting.

Otherwise, I think we could wind up with two lines, the one we insert and the line that would have already existed.. e.g. if the field isn't commented out (i.e. doesn't start with `#`), or has a value other than `yes`.

What do you think?